tests/functional/pod_and_daemons/test_mds_cache_trim_standby.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    brown_squad,
+    skipif_ocs_version,
+    bugzilla,
+)
+from ocs_ci.framework.testlib import E2ETest, tier2, skipif_external_mode
+from ocs_ci.helpers.e2e_helpers import run_metadata_io_with_cephfs
+from ocs_ci.ocs import cluster
+from ocs_ci.ocs.resources.pod import get_pod_logs
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2141422")
+@brown_squad
+@skipif_ocs_version("<4.15")
+@skipif_external_mode
+class TestMdsCacheTrimStandby(E2ETest):
+    @pytest.fixture(scope="function", autouse=True)
+    def teardown(self, request):
+        def finalizer():
+            """
+            This function will call a function to clear the mds memory usage gradually
+
+            """
+            cluster.bring_down_mds_memory_usage_gradually()
+
+        request.addfinalizer(finalizer)
+
+    @pytest.mark.polarion_id("OCS-6280")
+    def test_mds_cache_trim_on_standby_replay(self, dc_pod_factory):
+        """
+        Verifies whether the MDS cache is trimmed or not in standby-replay mode.
+
+        """
+        run_metadata_io_with_cephfs(dc_pod_factory, no_of_io_pods=5)
+        log.info(
+            "Starting metadata IO in the background. Monitoring for MDS cache alerts."
+        )
+
+        trim_msgs = ["cache trim"]
+        cache_warning = "MDSs report oversized cache"
+
+        for sampler in TimeoutSampler(
+            timeout=1800,
+            sleep=20,
+            func=cluster.get_active_mds_memory_utilisation_in_percentage,
+        ):
+            if sampler > 75:
+                break
+            else:
+                log.warning("MDS memory consumption is not yet reached target")
+
```

Comment:
How are you making sure cache utilization is maximum?

Reply:
Previously, the MDS Cache oversized warning was triggered at 75%  allocated memory utilization [which means that 150% of cache utilisation]. If such a warning appears, it should now only occur after the memory utilization exceeds 75%.